packages/accounts-password/password_server.js

```diff
@@ -573,119 +573,187 @@ Accounts.sendResetPasswordEmail = function (userId, email) {
     handleError("No such email for user.");
   }
 
-  var token = Random.secret();
-  var when = new Date();
   var tokenRecord = {
-    token: token,
+    token: Random.secret(),
     email: email,
-    when: when,
-    reason: 'reset'
-  };
-  Meteor.users.update(userId, {$set: {
-    "services.password.reset": tokenRecord
-  }});
-  // before passing to template, update user object with new token
-  Meteor._ensure(user, 'services', 'password').reset = tokenRecord;
-
-  var resetPasswordUrl = Accounts.urls.resetPassword(token);
-
-  var options = {
-    to: email,
-    from: Accounts.emailTemplates.resetPassword.from
-      ? Accounts.emailTemplates.resetPassword.from(user)
-      : Accounts.emailTemplates.from,
-    subject: Accounts.emailTemplates.resetPassword.subject(user)
+    when: new Date()
   };
 
-  if (typeof Accounts.emailTemplates.resetPassword.text === 'function') {
-    options.text =
-      Accounts.emailTemplates.resetPassword.text(user, resetPasswordUrl);
+  if (reason === 'resetPassword') {
+    tokenRecord.reason = 'reset';
   }
-
-  if (typeof Accounts.emailTemplates.resetPassword.html === 'function') {
-    options.html =
-      Accounts.emailTemplates.resetPassword.html(user, resetPasswordUrl);
+  else if (reason === 'enrollAccount') {
```

Comment:
The `else if`'s here could be moved up a line to sit beside the closing brace, to match the style used in the rest of the file.